pkg/kubelet/dockertools/manager.go

```diff
@@ -655,12 +665,25 @@ func (dm *DockerManager) runContainer(
 	return dockerContainer.ID, nil
 }
 
-func setEntrypointAndCommand(container *api.Container, opts *docker.CreateContainerOptions) {
+func setEntrypointAndCommand(container *api.Container, opts *kubecontainer.RunContainerOptions, dockerOpts *docker.CreateContainerOptions) {
```

Comment:
I assume that Rocket is going to need a version of this as well? The current way this function is laid out doesn't seem to lend itself to reuse for Rocket